README.md

```diff
@@ -1,5 +1,20 @@
 # Xamarin.iOS & Xamarin.Mac #
 
+|              | Status                                    |
+|--------------|-------------------------------------------|
+| master       | [![xamarin-macios-builds-master][1]][2]   |
+| xcode8.2     | [![xamarin-macios-builds-xcode8.2][3]][4] |
+| pr-builder   | [![xamarin-macios-pr-builder][5]][6]      |
```

Comment:
Showing the status of the PR job in the readme is not very useful in my experience. If it's red you typically can't do much about it anyway, e.g. someone had compile errors in a PR or whatever causes the build to fail.